filters/ratelimit/leakybucket.go

```diff
@@ -0,0 +1,113 @@
+package ratelimit
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/ratelimit"
+)
+
+type leakyBucket interface {
+	Add(ctx context.Context, label string, increment int) (added bool, retry time.Duration, err error)
+}
+
+type leakyBucketSpec struct {
+	create func(capacity int, emission time.Duration) leakyBucket
+}
+
+type leakyBucketFilter struct {
+	label     *eskip.Template
+	bucket    leakyBucket
+	increment int
+}
+
+func NewLeakyBucket(registry *ratelimit.Registry) filters.Spec {
+	return &leakyBucketSpec{
+		create: func(capacity int, emission time.Duration) leakyBucket {
+			return ratelimit.NewClusterLeakyBucket(registry, capacity, emission)
+		},
+	}
+}
+
+func (s *leakyBucketSpec) Name() string {
+	return filters.ClusterLeakyBucketRatelimitName
+}
+
+// clusterLeakyBucketRatelimit("a-label-${template}", leakVolume, "leak period", capacity, increment)
```

Comment:
godoc compliant would be:
// CreateFilter ...

Reply:
Moved the doc to the spec constructor like it is for other filters